db/seeds/test_subscribers.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex("subscribers").del()
+    .then(function () {
+      // Inserts seed entries
+      return knex("subscribers").insert([
+        {
+          sha1: "e76f6c9da0e88b888a98f896b25ee4dcbf769579",
```

Comment:
Do we want to hardcode the SHA1 of the emails, or just use the `getSha1()` method to keep everything in sync??

Reply:
Updated.